symphony-bdk-core/src/test/java/com/symphony/bdk/core/config/BdkConfigLoaderTest.java

```diff
@@ -1,53 +1,79 @@
 package com.symphony.bdk.core.config;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
 import com.symphony.bdk.core.config.model.BdkConfig;
-import com.symphony.bdk.core.exceptions.BdkConfigException;
+import com.symphony.bdk.core.config.exceptions.BdkConfigException;
 import org.junit.jupiter.api.Test;
 
 import java.io.InputStream;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class BdkConfigLoaderTest {
 
     @Test
-    public void loadFromYamlInputStreamTest() throws JsonProcessingException, BdkConfigException {
-        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream("/config.yaml");
+    public void loadFromYamlInputStreamTest() throws BdkConfigException {
+        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream("/config/config.yaml");
         BdkConfig config = BdkConfigLoader.loadFromInputStream(inputStream);
         assertEquals(config.getBot().getUsername(), "tibot");
     }
 
     @Test
-    public void loadFromJsonInputStreamTest() throws JsonProcessingException, BdkConfigException {
-        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream("/config.json");
+    public void loadFromJsonInputStreamTest() throws BdkConfigException {
+        InputStream inputStream = BdkConfigLoaderTest.class.getResourceAsStream("/config/config.json");
         BdkConfig config = BdkConfigLoader.loadFromInputStream(inputStream);
         assertEquals(config.getBot().getUsername(), "tibot");
     }
 
     @Test
-    public void loadFromYamlFileTest() throws JsonProcessingException, BdkConfigException {
-        String configPath = System.getProperty("user.dir") + "/src/test/resources/config.yaml";
+    public void loadFromYamlFileTest() throws BdkConfigException {
+        String configPath = System.getProperty("user.dir") + "/src/test/resources/config/config.yaml";
```

Comment:
I'm not sure that this is the safest way to locate the test resources. I'd personally use another strategy :
- using https://www.baeldung.com/junit-5-temporary-directory#1-tempdir-as-a-method-parameter
- copy the config file from the classpath to tempdir
- read config from tempdir

WDYT?

Reply:
It's a good idea! I will try that way. 👍